Curs04/src/views/evaluation.less

```diff
@@ -0,0 +1,266 @@
+//Evaluation less file
+
+//Import colors file
+@import 'colors';
+
+.logout-button {
+    float: right;
+    padding-top: 15px;
+}
+
+.evaluations-link {
+    margin-top: 15px;
+}
+
+.header-links {
+    display: inline-flex;
+}
+
+.container-section {
+    margin-top: 200px;
+}
+
+.table-section {
+    width: 70%;
+    margin: 0 auto;
+}
+
+.navigation-bar {
+    border-bottom: 2px solid @darkgrey;
+    padding-bottom: 5px;
+    float: right;
+}
+
+.left_image {
+    height: 290px;
+}
+
+.right_image {
+    float: right;
+}
+
+.right_description {
+    font-weight: bold;
+    margin-top: 50px;
+}
+
+.container-fluid {
+    background-color: lightgrey;
+}
+
+a {
+    text-decoration: none;
+    cursor: pointer;
+    color: black;
+    font-weight: 700;
+    font-size: 24px;
+}
+
+th,
+td {
+    padding-left: 55px;
+    font-size: 20px;
+}
+
+td {
+    border-bottom: 1px solid black;
+}
+
+/*Footer section*/
+.footer-section {
+    position: fixed;
+    right: 0;
+    bottom: 0;
+    left: 0;
+}
+
+table tr th {
+    background-color: #01a8f4;
+    color: white;
+    padding: 10px 0;
+}
+
+.logo-image {
+    width: 15%;
+    padding-right: 130px;
+}
+
+.footer-description {
+    font-weight: bold;
+    font-size: 16px;
+    color: @grey;
+}
+
+.container_table {
+    margin-top: 120px;
+}
+
+.evaluation-button {
+    padding-top: 4px;
+}
+
+.new-evaluation {
+    margin-left: 100px;
+}
+
+a.border-bottom:hover {
+    border-bottom: 4px solid @blue;
+    border-bottom-width: 7px;
+}
+
+table {
+    width: 100%;
+}
+
+/*Evaluation html*/
+
+.add-button {
+    background-color: @lightBlue;
+    margin: 8px 0;
+    color: white;
+    height: 100%;
+    font-size: 20px;
+    border-radius: 100%;
+    border: 1px solid @lightBlue;
+}
+
+/* Large screens ----------- */
+@media screen and (max-width: 1824px) and (min-width: 1200px) {
```

Comment:
for readability it is usually `min-width` first si `max-width` last

Reply:
👍